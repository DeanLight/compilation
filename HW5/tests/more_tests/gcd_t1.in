int gcd(int a,int b)
{
	if(a<b)
		return gcd(b,a);
	if (b==0)
		return a;
	int div = a/b; // todo ask
	int mod = a - (div*b);
	return gcd(b,mod);
}

void main()
{
	printi(gcd(121,33));
}