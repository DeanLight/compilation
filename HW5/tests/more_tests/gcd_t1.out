Loaded: ./exceptions.s
11